_posts/2021-09-01-cockpit-252.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Cockpit 252
+author: skobyda
+date: '2021-09-01'
+tags: cockpit
+slug: cockpit-252
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 252:
+
+
+## webserver: Drop remotectl utility
+
+[Cockpit 241](https://cockpit-project.org/blog/cockpit-241.html) already stopped using the deprecated `remotectl` utility for creating certificates for the main `cockpit.service`, in favor of the much simpler `/usr/libexec/cockpit-certificate-ensure`. This version moves the remaining usage (containers and documentation) to that helper, and drops `remotectl`.
+
+If you use scripts, playbooks, or similar direct invocations of `remotectl`, please consider removing that: Cockpit creates or refreshes a self-signed certificate on startup automatically if required. If you must pre-create it, please move your scripts to call
+`/usr/libexec/cockpit-certificate-ensure`. However, please note that this script is *not* a stable and supported public interface.
+
+## shell: Show package version in 'About web console' modal
```

Comment:
Let's capitalize "shell" for consistency, and say version*s*. Please adjust in the PR and use the script to regenerate the blog, to keep these two  in sync for as long as possible/desirable.